scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/solver/PrioritySolver.java

```diff
@@ -609,7 +617,7 @@ private Optional<MissingActivityNetworkConflict> makeActivityNetworkConflict(
         curPreviouslyInstantiated = null;
       }
       taskNetwork.startsAfterStart(
-          allActivitiesInNetwork.getLast(),
```

Comment:
why did you change this?

Reply:
IntelliJ was getting very upset with me using `getLast()` and `getFirst()`, saying it wasn't supported on `List`. Checking the `List` interface supported this assertion, as did [this](https://stackoverflow.com/questions/1049518/why-does-the-java-list-interface-not-support-getlast). I'm pretty skeptical but this was the only way I could compile!

Reply:
It is supported in Java 21, so I think the problem must be in your setup, not the code. Please revert this.

On Mac, you can go to `File > Project Structure > Project > SDK` and to change your jdk version.